share/esmf_wrf_timemgr/ESMF_TimeMod.F90

```diff
@@ -1080,16 +1080,20 @@ function ESMF_TimeDiff(time1, time2)
 
       ! Can either be yr/month based diff if diff is only in year and month
       ! or absolute seconds if diff in day/seconds as well
-
-      if (d1 == d2 .and. s1 == s2) then
-!         write(6,*) 'tcx timedifft ym'
-         ESMF_TimeDiff%YR = y1 - y2
-         ESMF_TimeDiff%MM = m1 - m2
-         cmptime%S  = 0
-         cmptime%Sn = 0
-         cmptime%Sd = 0
-         ESMF_TimeDiff%basetime = cmptime
-      else
+      !
```

Comment:
I am concerned about bringing this change in. We need to get a review from Tony who originally implemented this as to what effect this will have in CESM.